tools/release-checks.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+APPDIR=Simplenote
+
+function pOk() {
+  echo "[$(tput setaf 2)OK$(tput sgr0)]"
+}
+
+function pFail() {
+  echo "[$(tput setaf 1)KO$(tput sgr0)]"
+}
+
+function printVersion() {
+  gradle_version=$(grep -E 'version ".*"' $APPDIR/build.gradle | sed s/version// | grep -Eo "[a-zA-Z0-9.-]+" )
+  echo "$APPDIR/build.gradle version $gradle_version"
+}
+
+function checkFileAgainstHash() {
+  filename=$1
+  known_checksum=$2
+  checksum=`sha1sum "$filename" | cut -d" " -f1`
+  if [ x$checksum != x$known_checksum ]; then
+    pFail
+    exit 6
+  fi
+  pOk
+}
+
+function checkGradleProperties() {
+  /bin/echo -n "Check gradle.properties..."
+  checkFileAgainstHash $APPDIR/gradle.properties 2e36d6696bc71af3bc976b838966ac40cf06e4fe
+}
+
+function checkKeystore() {
+  keystore=`cat gradle.properties | grep storeFile | cut -d= -f 2 | sed -e 's/^[ ]*//'`
```

Comment:
Are we meant to run this from the root app directory? If so I needed to change this to: `cat $APPDIR/gradle.properties`